patch-diff-report-tool/config/import-control.xml

```diff
@@ -15,6 +15,9 @@
         <allow pkg="com.github.checkstyle.parser.StaxContentParser" />
         <allow pkg="com.github.checkstyle.PreparationUtils" />
         <allow class="org.thymeleaf.util.StringUtils" />
+        <file name="Statistics">
+            <allow class="java.math.BigInteger"/>
```

Comment:
please allow `java.math.BigInteger` new package in whole `data`.
testing tool should not be over strict on imports, we just enforced such Check to keep same style in project.